GrandCentralBoard/Widgets/Bonus/Extensions/NSDate.swift

```diff
@@ -8,22 +8,22 @@ import Foundation
 
 extension NSDate: Comparable { }
 
-public func ==(lhs: NSDate, rhs: NSDate) -> Bool {
+public func == (lhs: NSDate, rhs: NSDate) -> Bool {
```

Comment:
I thought we decided not to put a space before the opening paren for functions/methods. Is there an exception for operators?